src/main/resources/templates/db_scripts/dropLegacyItemEffectiveLocationFunctions.sql

```diff
@@ -0,0 +1,2 @@
+DROP FUNCTION IF EXISTS ${myuniversity}_${mymodule}.update_effective_location_on_holding_update CASCADE;
+DROP FUNCTION IF EXISTS ${myuniversity}_${mymodule}.update_effective_location_on_item_update CASCADE;
```

Comment:
Why are these using `CASCADE`?

Reply:
I found that other scripts in this module that were dropping database functions were doing it using cascade.  It seemed odd, but I wasn't sure if there was a reason I wasn't aware of.

Reply:
I suggest we don't do that. 

I'd rather it fail if there were unexpected uses of these functions, rather than for those uses to be arbitrarily dropped as well.